Makefile

```diff
@@ -1,7 +1,17 @@
-REGISTRY       ?= quay.io
-ORG            ?= kubernetes_incubator
-TAG            ?= $(shell git describe --always --dirty)
-IMAGE          ?= $(REGISTRY)/$(ORG)/node-feature-discovery-operator:$(TAG)
+IMAGE_BUILD_CMD ?= docker build
+IMAGE_BUILD_EXTRA_OPTS ?=
+IMAGE_PUSH_CMD ?= docker push
+
+VERSION := $(shell git describe --tags --dirty --always)
+
+IMAGE_REGISTRY ?= k8s.gcr.io/nfd
+IMAGE_NAME ?= node-feature-discovery-operator
```

Comment:
Nit: I think IMAGE_NAME should be initialized with `:=`

Reply:
fixed

Reply:
Is it??

Reply:
No? damm didn't got into the `git add` one sec

Reply:
how about now?